src/index.pug

```diff
@@ -7,6 +7,10 @@ html
       body {
         background-color: #eee !important;
       }
+    script.
```

Comment:
To the best of my knowledge `ProvidePlugin` can also do that. but sure, use which ever you find more fit. I just mean not to use script tags.